plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -161,8 +156,12 @@ class OptunaSweeperConf:
     _target_: str = "hydra_plugins.hydra_optuna_sweeper.optuna_sweeper.OptunaSweeper"
     defaults: List[Any] = field(default_factory=lambda: defaults)
 
-    optuna_config: OptunaConfig = OptunaConfig()
+    # Sampling algorithm
+    # Please refer to the reference for further details
+    # https://optuna.readthedocs.io/en/stable/reference/samplers.html
+    sampler: Any = MISSING
 
+    optuna_config: OptunaConfig = OptunaConfig()
```

Comment:
I think we can inline the content of OptunaConfig in this node to simplify the config further.
Maybe this can go into a followup diff as it's another breaking change. what do you think?